commit e191f9c5956dce30901c4416c9905327a419c40d
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2019-04-18T11:33:19-07:00 (6 months ago)
    
    OS-7758 test-vm fails due to out of quota instead of show write

diff --git a/src/vm/tests/test-update.js b/src/vm/tests/test-update.js
index 144c610d..7603f333 100644
--- a/src/vm/tests/test-update.js
+++ b/src/vm/tests/test-update.js
@@ -20,7 +20,7 @@
  *
  * CDDL HEADER END
  *
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  *
  */
 
@@ -1286,7 +1286,7 @@ test('fill up zoneroot', function (t) {
     execFile('/usr/bin/dd', [
         'if=/dev/zero',
         'of=/zones/' + vm_uuid + '/root/zeros',
-        'bs=1M'
+        'bs=37M'
     ], function (err, stdout, stderr) {
         var match = stderr.match(/dd: unexpected short write, wrote/);
         t.ok(match, 'expected short write'
